monkey/infection_monkey/monkey.py

```diff
@@ -242,7 +242,7 @@ def _subscribe_events(
     def _get_local_network_interfaces():
         local_network_interfaces = get_local_network_interfaces()
         for i in local_network_interfaces:
-            logger.debug(f"Found local interface {i.address}{i.netmask}")
+            logger.debug(f"Found local interface {i.ip.compressed}/{i.network.prefixlen}")
```

Comment:
I would prefer `str(interface.ip)` to `interface.ip.compressed` or `interface.ip.exploded`. What does "compressed" mean? How is it different from "exploded"?